README.md

```diff
@@ -13,7 +13,7 @@
 
 ml5.js aims to make machine learning approachable for a broad audience of artists, creative coders, and students. The library provides access to machine learning algorithms and models in the browser, building on top of [TensorFlow.js](https://js.tensorflow.org/).
 
-The library is supported by code examples, tutorials, and sample data sets with an emphasis on ethical computing. Bias in data, stereotypical harms, and responsible crowdsourcing are part of the documentation around data collection and usage.
+The library is supported by code examples, tutorials, and sample data sets with an emphasis on ethical computing. Bias in data, stereotypical harms, and responsible crowdsourcing are part of the documentation around data collection and usage.Also, if you have any doubts you can personally ask any member of your generous community on [Github](https://github.com/ml5js/ml5-library/).
```

Comment:
Hmm. Thanks for adding this. I'm not sure this is quite the right place for this piece of text. May we please remove this addition from the PR?

Reply:
I think the sentiment is correct -- we are supportive and happy to be as helpful as we can -- but I don't think it needs to be covered here. 

+1 to your point @bomanimc // I think this is resolved in the Code of Conduct documentation and we should remove this.